src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/CategorySyncIT.java

```diff
@@ -16,6 +16,7 @@
 import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 
 import java.util.ArrayList;
```

Comment:
Lines 162-174 of this file:

https://github.com/commercetools/commercetools-sync-java/blob/82a1e55c87a4dbab0259f57fda5a05a830034ee1/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/CategorySyncIT.java#L162-L174

Reply:
Not sure why it doesn't fail =/

Reply:
because it is ignored with the @Ignored annotation. I leave it there on purpose because it should be fixed on addressing issue #138. Check todo comment: https://github.com/commercetools/commercetools-sync-java/blob/82a1e55c87a4dbab0259f57fda5a05a830034ee1/src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/CategorySyncIT.java#L144